src/openfermion/transforms/_unitary_rotations.py

```diff
@@ -0,0 +1,33 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This seems a little bit light to be its own module, unless you plan to build more. Perhaps there is some other place it might fit?

Reply:
Perhaps it can be combined with _projection.py ? Both files are used for manipulating QubitOperator objects, although in slightly different manners.

Reply:
equality_constraints_projection? I think that's probably not a good place for it. Since this is used for measurements, perhaps I'd feel better if you put it in measurements/_unitary_rotations.py. Does that sound okay?

Reply:
Oh okay. Either way works for me. transforms/_projections.py would be okay too.

Reply:
Ok, I'll combine this with transforms/_projections.py then.